spec/unit/packagers/rpm_spec.rb

```diff
@@ -217,6 +218,7 @@ module Omnibus
           expect(contents).to include("/.file1")
           expect(contents).to include("%dir /dir2")
           expect(contents).to include("/file2")
+          expect(contents).to include("%dir \"/dir3 space\"")
```

Comment:
I stared at this too when I was adding the test.  Because of the space, the path name `/dir3 space` needs to be quoted in the spec file.  The problem I fixed was that we were taking the path name as including the double quotes, e.g. `"/dir3 space"`. Since that isn't the path name of an actual file, we didn't detect it as such and thus didn't mark the entry corresponding to the path `/dir3 space` as a directory in the spec file.